superset/views/core.py

```diff
@@ -568,6 +568,7 @@ def annotation_json(  # pylint: disable=no-self-use
     @permission_name("explore_json")
     @expose("/explore_json/data/<cache_key>", methods=("GET",))
     @check_resource_permissions(check_explore_cache_perms)
+    @deprecated(eol_version="3.0")
```

Comment:
I could't find any documentation on "eol_version", only "version". Is "eol_version" when it was first deprecated or when it is planned to be removed?

Reply:
I understand it's when it's getting removed (end-of-life)